src/solidity/ethpmresolver.ts

```diff
@@ -1,28 +1,41 @@
-import * as path from "path";
+import Debug from "debug";
 import { ResolverContext, SubResolver } from "../resolvers";
-import { BacktrackFsResolver } from "../resolvers/backtracktfsresolver";
+import { BacktrackFsResolver } from "../resolvers/backtrackfsresolver";
+const debug = Debug("resolverengine:ethpmresolver");
 
 // 1st group - package name
 // 2nd group - contract path
-const FILE_LOCATION_REGEX = /^(.+?)\/(.+)$/;
+const FILE_LOCATION_REGEX = /^([^/]+)\/(.+)$/;
 
-export function EthPmResolver(epmDirectory: string = "installed_contracts/"): SubResolver {
-  const fsResolver = BacktrackFsResolver(epmDirectory);
+const prefixT = "installed_contracts";
```

Comment:
Naming

Reply:
I didn't want to call Truffle by name, can I?